include/translatable-object.php

```diff
@@ -508,4 +498,21 @@ public function set_language_in_mass( $ids, $lang ) {
 		// Invalidate our cache.
 		wp_cache_set( 'last_changed', microtime(), $this->cache_type );
 	}
+
+	/**
+	 * Returns database-related informations that can be used in some of this class methods.
+	 * These are specific to the table containing the objects.
+	 *
+	 * @since 3.4.3
+	 * @see PLL_Translatable_Object::join_clause()
+	 * @see PLL_Translatable_Object::get_objects_with_no_lang_sql()
```

Comment:
Just a formatting preference.
```suggestion
	 * @see PLL_Translatable_Object::join_clause()
	 * @see PLL_Translatable_Object::get_objects_with_no_lang_sql()
	 *
	 * @since 3.4.3
```

Reply:
Done in [6c26463](https://github.com/polylang/polylang/pull/1301/commits/6c26463fa50a64453e20d07fc31b2381963f6842).